balloc/balloc.c

```diff
@@ -158,59 +157,62 @@ static bool is_normal(uint64_t alloc_flags);
 static bool is_any(uint64_t alloc_flag);
 
 
-/* This macro is to control the debug verbose message */
-#undef BALLOC_ENABLE_DUMP
-
 static void balloc_debug_dump_extent(const char *tag, struct m0_ext *ex)
 {
-#ifdef BALLOC_ENABLE_DUMP
+if (ENABLE_BALLOC_DUMP) {
 
 	if (ex == NULL)
 		return;
 
-	M0_LOG(M0_DEBUG, "dumping ex@%p:%s\n"
-	       "|----"EXT_F, ex, (char*) tag, EXT_P(ex));
-#endif
+	M0_LOG(M0_DEBUG, "ex@%p:%s "EXT_F, ex, (char*) tag, EXT_P(ex));
+}
 }
 
 M0_INTERNAL void m0_balloc_debug_dump_group(const char *tag,
 					    struct m0_balloc_group_info *grp)
 {
-#ifdef BALLOC_ENABLE_DUMP
+if (ENABLE_BALLOC_DUMP) {
```

Comment:
Thank you to take care of this piece of code. It is already outdated.
Otherwise, it will not be updated.